package.json

```diff
@@ -38,6 +40,7 @@
     "qrcode.react": "1.0.1",
     "raw-loader": "4.0.2",
     "react": "16.13.1",
+    "react-copy-to-clipboard": "5.0.3",
```

Comment:
We already copy text to the clipboard (TokenList.jsx) using built-in functionality, so don't need this extra dependency

Reply:
Done.